spec/controllers/gsis_controller_spec.rb

```diff
@@ -127,16 +141,38 @@ def update_and_check_if_hired(email, gsi_amount_change)
           gsi.save!
         end
 
-        describe "if the new GSI doesn't exist" do
-          it 'creates the new GSI and deletes the old one' do
-            update_and_check_if_hired(email, 0)
+        describe 'if the updated GSI has other appointments' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.